master/internal/checkpoint_gc.go

```diff
@@ -73,6 +73,11 @@ func newCheckpointGCTask(
 func (t *checkpointGCTask) Receive(ctx *actor.Context) error {
 	switch msg := ctx.Message().(type) {
 	case actor.PreStart:
+		if t.ToDelete == "" && !t.DeleteTensorboards {
+			// Early return as nothing to do
+			t.completeTask(ctx)
```

Comment:
no need for completeTask since `AddTask` hasn't even run yet. we should just call `ctx.Self().Stop()` and return nil.